Brixel/Singer
# Feature/ab#131

To test:
1. Change a registration (location, approval)
2. Visit the 'te verwerken-emails' page
3. Review the contents
4. Send emails
5. Check if the email are correct
